### PR TITLE
docs: unify documentation

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

- Update main Readme.md
- Update License and authors :trollface: 
- Move instruction of using `web-component` to `./docs/usage/web-component.md` document.
- Move instruction of using `web-component` in Angular to `./docs/usage/angular.md` document.
- Describe how to use the component in NextJS app in `./docs/usage/nextjs.md` document.
- Mention about `esm`, `cjs` and `umd` modules.

**Related issue(s)**
Part of https://github.com/asyncapi/shape-up-process/issues/83

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,24 +160,10 @@ If you have a bug to report, reproduce it in an online code editor. For example,
...
 ## Missing features
 
-See the list of features that are still missing in the component:
-
-- [ ] render [`bindings`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#fixed-fields-19)
-- [ ] render [`tags`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#tagsObject)
-- [ ] render [`externalDocs`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#externalDocumentationObject)
-- [ ] render [`schema properties`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#properties) 
-- [ ] render [`correlationID`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#correlationIdObject)
+To see features we don't support, please check [issues](https://github.com/asyncapi/asyncapi-react/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
```
Comment (derberg) [inline README.md:163]: not all enhancements are about adding missing features. In my opinion better just remove this section

File: docs/configuration/config-modification.md [hunk 2]
```diff
@@ -82,19 +47,19 @@ See exemplary component configuration in TypeScript and JavaScript.
 ```tsx
 import * as React from "react";
 import { render } from "react-dom";
-import AsyncApiComponent, { ConfigInterface } from "asyncapi-react";
+import AsyncApiUI, { ConfigInterface } from "@asyncapi/react-component";
```
Comment (derberg) [inline docs/configuration/config-modification.md:50]: why AsyncAPIUI? it is AsyncAPIComponent after all

File: docs/development/guide.md [hunk 3]
```diff
@@ -79,13 +81,11 @@ See the rules for naming functions and components, and the architecture conventi
 - Each line should contain no more than one statement.
 - Always put the opening brace in the same line as the previous statement.
 - Avoid using the ternary operator (`statement ? true : false`). Use the guard (`&&`) instead.
-- Do not use a prefix or a suffix on an interface like `IVariable` or `VariableInterface`. That does not apply to `AsyncApiInterface`, `ConfigInterface`, and `ThemeInterface`.
+- Do not use a prefix or a suffix on an interface like `IVariable` or `VariableInterface`. That does not apply to `AsyncApiInterface` and `ConfigInterface`.
```
Comment (derberg) [inline docs/development/guide.md:84]: I thought `AsyncApiInterface` goes away. At least you removed it from main readme

File: README.md [hunk 4]
```diff
@@ -52,13 +51,64 @@ Check out this simple sandbox application that uses the React component:
...
+    errors: false,
+  },
+};
+
+const App = () => <AsyncApiUI schema={schema} config={config} />;
+
+render(<App />, document.getElementById("root"));
+```
+
+### Another examples
```
Comment (derberg) [inline README.md:97]:
```suggestion
### More examples
```

File: README.md [hunk 5]
```diff
@@ -52,13 +51,64 @@ Check out this simple sandbox application that uses the React component:
 
 [![Edit asyncapi-react-component-in-action](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/asyncapi-react-component-in-action-wvdy2)
 
-### Props
+## Usage
+
```
Comment (derberg) [inline README.md:55]:
```suggestion
### Simple example in React
```

File: docs/usage/web-component.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+```sh
+npm install --save @asyncapi/web-component
+```
+
+Check out this simple sandbox application that uses the Web Component in Angular project:
+
+[![Edit asyncapi-web-component-in-action](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/asyncapi-web-component-in-action-l652x)
+
+## Usage
```
Comment (derberg) [inline docs/usage/web-component.md:21]:
```suggestion
## Examples
```

sometimes you write examples sometimes usage, stick to one

File: README.md [hunk 7]
```diff
@@ -52,13 +51,64 @@ Check out this simple sandbox application that uses the React component:
 
 [![Edit asyncapi-react-component-in-action](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/asyncapi-react-component-in-action-wvdy2)
 
-### Props
+## Usage
```
Comment (derberg) [inline README.md:54]:
```suggestion
## Examples
```

or just two sections `##`
- Using in React
- Using in other technologies

File: README.md [hunk 8]
```diff
@@ -197,24 +160,10 @@ If you have a bug to report, reproduce it in an online code editor. For example,
...
-- [ ] render [`bindings`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#fixed-fields-19)
-- [ ] render [`tags`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#tagsObject)
-- [ ] render [`externalDocs`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#externalDocumentationObject)
-- [ ] render [`schema properties`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#properties) 
-- [ ] render [`correlationID`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#correlationIdObject)
+To see features we don't support, please check [issues](https://github.com/asyncapi/asyncapi-react/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
 
 If you want to help us develop them, feel free to contribute.
 
-## Credits
```
Comment (derberg) [inline README.md:210]: I don't think it is fair to wipe out history of the project and not remember who donated the project. We do not need huge logo as nobody from Kyma participates in project development, but we still should keep information about initial author, here or in the package.json

**Review by derberg — CHANGES_REQUESTED** [review]
looking on the change, IMHO we should release it as 1.0.0-rc.1

there are breaking changes that will cause issues to users if they just automatically bump. Component is changing a lot and we should clearly indicate that. This will also enable us to ship something that is not final and get feedback from current users.

We also need a migration guide, a short list of things that users need to do to switch to this release candidate. Best if it is published under the releasae

File: README.md (hunk 1 above)
Comment (magicmatatjahu) [reply README.md:163]: You're right.

File: docs/configuration/config-modification.md (hunk 2 above)
Comment (magicmatatjahu) [reply docs/configuration/config-modification.md:50]: Maybe we rename it to `@asyncapi/ui` -> https://github.com/asyncapi/asyncapi-react/issues/263, but yeah, I will leave it as `AsyncApiComponent`.

File: docs/development/guide.md (hunk 3 above)
Comment (magicmatatjahu) [reply docs/development/guide.md:84]: Good catch! I forgot about that 😅

File: docs/usage/web-component.md (hunk 6 above)
Comment (magicmatatjahu) [reply docs/usage/web-component.md:21]: I prefer `Usage` :)

File: README.md (hunk 8 above)
Comment (magicmatatjahu) [reply README.md:210]: Ok, I will leave the info about Kyma project, but without logo.

**Comment by magicmatatjahu** [comment]
@derberg Thanks for review!

> looking on the change, IMHO we should release it as 1.0.0-rc.1
>
> there are breaking changes that will cause issues to users if they just automatically bump. Component is changing a lot and we should clearly indicate that. This will also enable us to ship something that is not final and get feedback from current users.

I don't agree with this `1.0.0-rc.1`. It should be released as `0.24.0` (with `0.23.0` where we fix all security issues in old version). Maybe the component has changed a lot, mainly the appearance, but I don't think that's a reason to start with `1.0.0` (even rc). We still have 0.X.X so users should be prepared for breaking change.

> We also need a migration guide, a short list of things that users need to do...

Sure :) I will remember about that. 

@fmvilas What do you think about release?

File: README.md [hunk 9]
```diff
@@ -195,25 +158,9 @@ If you create a feature request, use the dedicated **Feature request** issue tem
...
-If you want to help us develop them, feel free to contribute.
-
 ## Credits
 
-<p align="center">
- <a href="https://kyma-project.io/" target="_blank">
-  <img src="https://raw.githubusercontent.com/kyma-project/kyma/master/logo.png" width="235">
- </a>
-</p>
+The project was originally developed under the [Kyma project](https://kyma-project.io/), in 2020 it was transferred to the AsyncAPI Initiative.
```
Comment (derberg) [inline README.md:163]: 2019 😉 
I know, time flies so fast, right? 😄

Comment (magicmatatjahu) [reply README.md:163]: I wrote year when Kyma give AsyncAPI 100% control of project :) I don't know if we need to clarify it.

Comment (derberg) [reply README.md:163]: you write transferred, transferred/moved/handed over, whatever word you choose, the right date is 2019. Unless you write `in 2020 it was handed over under maintenance of the AsyncAPI Initiative` or `in 2019 it was moved under AsyncAPI Initiative`. Still think the 2nd one is better, but I care more about wording

Comment (magicmatatjahu) [reply README.md]: I changed to `in 2019 it was moved under AsyncAPI Initiative` :) Thanks!

**Review by derberg — APPROVED** [review]
I approve. Topic of release is not blocking for this PR imho. Just open another issue or simply make a note that in a final PR we need to discuss the release type and migration guide

**Comment by magicmatatjahu** [comment]
@derberg Thanks! I will add note in final PR.

**Pull request merged**